### PR TITLE
Adds implementation for E

### PR DESCRIPTION
**Pull request by EiffL**
This PR is adding an implementation coming from the development notebooks.

### CODE REVIEW

**Comment by EiffL** [comment]
When this PR is merged, it will solve the E function task in #46

File: flowpm/tfbackground.py [hunk 1]
```diff
@@ -19,7 +19,29 @@ def w(cosmo,a):
...
+  Returns
+  -------
+  E(a): array
+    Scale factor dependent factor E(a)
+
+  Notes
+  -----
+  The Hubble parameter at scale factor `a` is given by
+  :math:`H^2(a) = E^2(a) H_o^2` where :math:`E^2` is obtained through
+  Friedman's Equation (see :cite:`2005:Percival`) :
```
Comment (dlanzieri) [inline flowpm/tfbackground.py:40]: please add more details about the reference

**Review by dlanzieri — APPROVED** [review]
thanks it looks good to me.

**Comment by EiffL** [comment]
Thjanks for the review, but it looks this now obsolete, so I'm going to cloise that PR

**Pull request closed without merging**